components/locationpicker/default.htm

```diff
@@ -0,0 +1,25 @@
+{% if not selectedCountryId %}
+    <div class="form-group">
+        <label for="accountCountry">{{ trans('winter.location::lang.locationPicker.countryFieldName.title') }}</label>
+        {{ form_select_country(__SELF__.property('countryFieldName'), countryId, {
+            id: 'accountCountry',
+            class: 'form-control',
+            emptyOption: '',
```

Comment:
Why remove the emptyOption value of "Select a x"?

Reply:
We'd have to translate that too. When it comes down to adding translate support, it might be better for people to override the partial and rely on the Translate plugin's CMS translation management.